roles/seqreports/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- name: create conda environment for seqreports
+  shell: "{{ anaconda_path }}/bin/conda create -n {{ seqreports_venv_name }} -c anaconda python=3.8 -y"
+  args:
+    creates: "{{ anaconda_path }}/envs/{{ seqreports_venv_name }}/bin/python3"
+
+- name: Set host fact boolean for miarka deployment
+  set_fact:
+    apply_to_conda: "{{ apply_to_conda | default(true) }}"
+
+- name: ensure seqreports data path exists
+  file:
+    path: "{{ seqreports_data_path }}"
+    state: directory
+
+- name: ensure nextflow directories exist
+  file:
+    path: "{{ item }}"
+    state: directory
+  loop:
+    - "{{ seqreports_nxf_temp }}"
+    - "{{ seqreports_nxf_work }}"
+    - "{{ seqreports_nxf_logs }}"
```

Comment:
Perhaps @matrulda could anser this?

Reply:
The sequencing-report-service takes these location as input parameters, so in a way yes. The way it is designed right now I would say that it's only important that the `nxf_logs` directory has a dedicated location since the service creates a subfolder in that directory (named after the job ID) and then starts nextflow from there, putting the nextflow log in that directory. An alternative approach would be to start nextflow in the runfolder to be processed, but then we would need to update the service.

Reply:
ok, perhaps we can let this be for now and think about it once we see the full seqreports role and how it fits into the grander scheme

Reply:
I've added a ticket for this https://snpseq.atlassian.net/browse/DEVELOP-1898